.chloggen/fix-utf8-reading.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: filelogreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Do not replace invalid UTF-8 bytes with \uFFFD when reading utf8 input.
```

```suggestion
note: Do not replace invalid UTF-8 bytes with \uFFFD when reading UTF-8 input.
```
Nit: Just wanted to be consistent with how we style `UTF-8` in the note.